### PR TITLE
Sycamore Android Radio Answers from config

### PR DESCRIPTION
**Pull request by clementzach**
This PR uses the config files and survey history files to substitute semicolons and commas from config files

### CODE REVIEW

**Comment by clementzach** [comment]
(I'm going to refactor the code a bit tomorrow, so don't worry about reviewing for now)

File: forest/sycamore/base.py [hunk 1]
```diff
@@ -80,7 +84,7 @@ def compute_survey_stats(
     else:
         agg_data = aggregate_surveys_config(
             study_folder, config_path, tz_str, users, start_date,
-            end_date,  augment_with_answers
+            end_date,  augment_with_answers, history_path
```
Comment (hackdna) [inline forest/sycamore/base.py:87]: Extra space before `augment_with_answers`

File: forest/sycamore/common.py [hunk 2]
```diff
@@ -398,6 +399,10 @@ def aggregate_surveys_config(
             The first date of the survey data, in YYYY-MM-DD format
         time_end(str):
             The last date of the survey data, in YYYY-MM-DD format
+        history_path: Path to survey history file. If this is included, the
```
Comment (hackdna) [inline forest/sycamore/common.py:402]: Either style is acceptable but you should continue putting arg names on a separate line for consistency.

File: forest/sycamore/common.py [hunk 3]
```diff
@@ -532,6 +539,10 @@ def append_from_answers(
             The last date of the survey data, in YYYY-MM-DD format
         config_path(str):
             Filepath to survey config file (downloaded from Beiwe website)
+        history_path: Path to survey history file. If this is included, the
```
Comment (hackdna) [inline forest/sycamore/common.py:542]: Either style is acceptable but you should continue putting arg names on a separate line for consistency.

File: forest/sycamore/common.py [hunk 4]
```diff
@@ -752,18 +764,15 @@ def read_aggregate_answers_stream(
             The first date of the survey data, in YYYY-MM-DD format
         time_end(str):
             The last date of the survey data, in YYYY-MM-DD format
+        history_path: Path to survey history file. If this is included, the
```
Comment (hackdna) [inline forest/sycamore/common.py:767]: Either style is acceptable but you should continue putting arg names on a separate line for consistency.

File: forest/sycamore/common.py [hunk 5]
```diff
@@ -782,6 +791,82 @@ def read_aggregate_answers_stream(
...
+        # they will have ints in their answer field.
+        & (aggregated_data["answer"].apply(
+            lambda x: x.isdigit() if isinstance(x, str)
+            else True if isinstance(x, int)
+            else False)),
+        "answer"
+        # if their question type looks like this, it was made on an Android
+        # device. Radio button questions are the only ones with the integer
+        # instead of text (Also, avoiding any possible text outputs)
+    ].index
```
Comment (hackdna) [inline forest/sycamore/common.py:813]: This is a huge statement. It probably makes sense to simplify it.

**Review by hackdna — CHANGES_REQUESTED** [review]
FYI, you can change stage of a PR to draft if it's not ready for review yet
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request
Thank you!

File: forest/sycamore/common.py (hunk 2 above)
Comment (clementzach) [reply forest/sycamore/common.py:402]: Ah, didn't realize I was doing that. Thanks!

File: forest/sycamore/common.py (hunk 5 above)
Comment (clementzach) [reply forest/sycamore/common.py:822]: Hopefully what I did makes more sense...

**Pull request merged**